index.html

```diff
@@ -275,14 +288,14 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
         method="POST"
       >
         <div class="all-components">
-          <div class="tonic-content">
+          <div class="tonic-content-contact">
             <h2>Contact me</h2>
             <p>
               If you have an application you are intersted in developing, a
               feature that you need built or a project that needs codiing, I'd
               love to help with it.
             </p>
-            <div class="tonic-courses">
+            <div class="tonic-courses-about">
               <input
                 type="text"
                 placeholder="Mack donald"
```

Comment:
- [ ] Good job so far but your mobile design still needs some work to be done there. I examined your app using the browser dev tools and the layout of the contact section doesn't fit the mobile screen, please, Kindly fix the mobile version of the contact section. 
- Your design:
<img width="218" alt="image" src="https://user-images.githubusercontent.com/77190203/150510062-721eee0a-2ed3-44c0-afc5-6db3dc1f00bd.png">